src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreTargetGraph.cs

```diff
@@ -90,6 +90,16 @@ public static RestoreTargetGraph Create(IEnumerable<GraphNode<RemoteResolveResul
             return Create(RuntimeGraph.Empty, graphs, context, logger, framework, runtimeIdentifier: null);
         }
 
+        /// <summary>
+        /// Factory method
+        /// </summary>
+        /// <param name="runtimeGraph"></param>
```

Comment:
I don't think there's much value in having the name without a description.

Reply:
Added docs.